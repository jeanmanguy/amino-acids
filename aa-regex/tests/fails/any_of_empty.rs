use aa_regex::*;

fn main() {
    let _wrong = any_of!();
}
