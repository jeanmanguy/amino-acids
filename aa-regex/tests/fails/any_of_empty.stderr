error: no amino acid to add
 --> $DIR/any_of_empty.rs:4:18
  |
4 |     let _wrong = any_of!();
  |                  ^^^^^^^^^
  |
  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
